helm-charts/common/speecht5/templates/deployment.yaml

```
@@ -27,6 +27,35 @@ spec:
       {{- end }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if not (hasPrefix "/data/" .Values.TTS_MODEL_PATH) }}
+      initContainers:
+        - name: model-downloader
+          envFrom:
+            - configMapRef:
+                name: {{ include "speecht5.fullname" . }}-config
+          securityContext:
+            allowPrivilegeEscalation: false
+            readOnlyRootFilesystem: true
+            capabilities:
+              drop:
+              - ALL
+              add:
+              - DAC_OVERRIDE
+              - FOWNER
```

Based on quick testing, `FOWNER` cap is not needed neither for `chmod`, nor writing to dirs owned by other user IDs (than `root`), and my quick test seems to verify that:
```suggestion
            runAsUser: 0
            seccompProfile:
              type: RuntimeDefault
```

It would also be good to include here comment why this is done like this:
> To download data although host (parent) dir would be owned by another user ID

---

The reason to add CAP_FOWNER here is to deal with backward compatibility issues for those existing model directories downloaded by previously released helm chart. Otherwise, chmod +R would fail.